### DISCUSSION TITLE
Corrupted db files after 2.1.0

### DISCUSSION THREAD

**Discussion by djin921** [seerr-team/seerr#1129] [Q&A] [answered]
Unable to start Jellyseerr after upgrading to 2.1.0. Currently running in a docker container, Jellyfin is working as expected and seeing the following in the Jellyseer logs:

```
> jellyseerr@2.1.0 start /app
> NODE_ENV=production node dist/index.js
[info]: Commit Tag: $GIT_SHA 
[info]: Starting Overseerr version 2.1.0 
[debug][Settings Migrator]: Checking migration '0001_migrate_hostname.js'... 
[debug][Settings Migrator]: Checking migration '0002_migrate_apitokens.js'... 
[error][Jellyfin API]: Something went wrong while creating an API key from the Jellyfin server {"error":"Unauthorized"}
[error][Settings Migrator]: Error while running migration '0002_migrate_apitokens.js': Failed to create Jellyfin API token from admin account. Please check your network configuration or edit your settings.json by adding an 'apiKey' field inside of the 'jellyfin' section to fix this issue. 
[error][Settings Migrator]: A common cause for this error is a permission issue with your configuration folder, a network issue or a corrupted database. 
```

I took a look into the db folder and the files (db.sqlite3, db.sqlite3-shm, db.sqlite3-wal) are filled with garbage data (See screenshot below) What's the recommended action here? Delete all the files in the db folder or is there a better action here?

![garbage](https://github.com/user-attachments/assets/1f1ce458-4805-43b8-ada5-1fd80547466c)

**Comment by fallenbagel** [comment, marked as answer]
Its not a database corruption. Logs literally states what to do. To add a jellyfin apikey into the jellyfin block of SETTINGS.JSON.

And you cant ofcourse view a database through a text editor. It should be viewed using sqlite3. **Its not a text file**. settings.json is a text file. The migrations were for that. In the logs it states what to do. If you find it difficult, you can backup settings.json and then delete settings.json and go through the steps and you should have the apikey added with your old data.

**Comment by djin921** [comment]
Got it, thank you for the help! The error logging is very accurate, should've spent more time reading and testing. Thanks!
